tools/swift-format/Sources/Rules/CommentWhitespace.swift

```diff
@@ -11,5 +11,95 @@ import SwiftSyntax
 ///
 /// - SeeAlso: https://google.github.io/swift#horizontal-whitespace
 public final class CommentWhitespace: SyntaxFormatRule {
+  public override func visit (_ token: TokenSyntax) -> Syntax {
+    var pieces = [TriviaPiece]()
+    var validToken = token
+    var isInvalid = false
```

Comment:
I'd avoid using `isInvalid` because it's a bit generic - maybe something like `needsWhitespaceFix`?